docs/run/max-execution-time.mdx

```diff
@@ -6,7 +6,7 @@ description: Describes how long a Qiskit Runtime job or session can run.
 
 # Maximum execution time 
 
-To ensure fairness and help control costs, there is a maximum amount of time each Qiskit Runtime job and session can run. If a job exceeds this time limit, it is forcibly canceled and a `RuntimeJobMaxTimeoutError` exception is raised. If a session exceeds its limits, any queued jobs are cancelled but any jobs that are already running are not. 
+To ensure fairness and help control costs, there is a maximum amount of time each Qiskit Runtime job and session can run. If a job exceeds this time limit, it is forcibly canceled and a `RuntimeJobMaxTimeoutError` exception is raised. If a session exceeds its limits, any queued jobs are canceled but any jobs that are already running are not. 
```

Comment:
One last nit - since we no longer talk about session max time here, we should add a link to the page that does talk about session max time.